src/client/components/Graphs.tsx

```diff
@@ -56,7 +73,192 @@ class Graphs extends Component<{}, GraphsState> {
+              <Tooltip />
+              <Legend />
+              <Line
+                dot={false}
+                legendType="square"
+                yAxisId="left"
+                dataKey="data.fastestFee"
+                stroke="#0b84a5"
+                name="Fastest Fee"
+              />
+              <Line
+                dot={false}
+                legendType="square"
+                yAxisId="left"
+                dataKey="data.halfHourFee"
+                stroke="#f6c860"
+                name="Half Hour Fee"
+              />
+              <Line
+                dot={false}
+                legendType="square"
+                yAxisId="left"
+                dataKey="data.hourFee"
+                stroke="#ca472f"
+                name="Hour Fee"
+              />
+              <Line
+                dot={false}
+                legendType="square"
+                yAxisId="left"
+                dataKey="data.minimumFee"
+                stroke="#9dd865"
+                name="Minimum Fee"
+              />
+            </LineChart>
+          </ResponsiveContainer>
+          <h1 style={graphTitle}>Earn</h1>
+          <ResponsiveContainer width="95%" height={400}>
+            <LineChart
+              margin={{
+                top: 20,
+                right: 20,
+                bottom: 0,
+                left: 20
+              }}
+              data={dataParser(this.state.data, 'Earn')}
+            >
+              <CartesianGrid stroke="#f5f5f5" />
+              <XAxis dataKey="_id" />
+              <YAxis
+                yAxisId="left"
+                orientation="left"
+                stroke="#000000"
+                width={105}
+                label={{
+                  value: 'satoshis/byte',
+                  angle: -90,
+                  position: 'center'
+                }}
+              />
+              <Tooltip />
+              <Legend />
+              <Line
+                dot={false}
+                legendType="square"
+                yAxisId="left"
+                dataKey="data.sortedData.zeroToOne"
+                stroke="#0b84a5"
+                name="0-1"
+              />
+              <Line
+                dot={false}
+                legendType="square"
+                yAxisId="left"
+                dataKey="data.sortedData.oneToTwo"
+                stroke="#f6c860"
+                name="1-2"
+              />
+              <Line
+                dot={false}
+                legendType="square"
+                yAxisId="left"
+                dataKey="data.sortedData.twoToThree"
+                stroke="#ca472f"
+                name="2-3"
+              />
+              <Line
+                dot={false}
+                legendType="square"
+                yAxisId="left"
+                dataKey="data.sortedData.threeToTen"
+                stroke="#9dd865"
+                name="3-10"
+              />
+            </LineChart>
+          </ResponsiveContainer>
```

Comment:
A lot of repeating code for the 3 graphs. I would create a new Component specifically for the graph. Then you could pass the title, labels, and data as props.

Maybe call the new component `Graph` and rename this component to `MainScene`, or whatever you feel is appropriate.